css/style.css

```diff
@@ -0,0 +1,84 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Inter', sans-serif;
+}
+
+/* Navbar section */
+nav {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+}
+
+/* End Navbar section */
+
+/* Welcome Message */
+
+.welcome {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 24px;
+  text-align: center;
+}
+
+/* End Welcome Message */
+
+/* Icon Menu */
+
+.icon-menu {
+  margin: 32px;
+  width: 24px;
+  height: 24px;
+}
+
+/* End Icon Menu */
+
+.hero-sec {
+  background-image: url('/img/header-illsutration-mobile.svg');
+  background-size: cover;
+  background-position: top;
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] This section should use a `flexbox` to align its items, it might seem like it's not important right now but later it will help you greatly with responsivity, so please add that. 😁

Reply:
I have implemented this